src/components/DappBrowser/Homepage.tsx

```diff
@@ -57,6 +57,10 @@ export const Homepage = React.memo(function Homepage() {
 });
 
 const Trending = React.memo(function Trending({ goToUrl }: { goToUrl: (url: string) => void }) {
+  const { dapps } = useDapps();
+
+  const trendingDapps = dapps.filter(dapp => dapp.trending).slice(0, 10);
```

Comment:
```suggestion
  const trendingDapps = dapps.filter(dapp => dapp.trending).slice(0, 8);
```

Would lower this slightly for now since currently these get rendered once for every homepage — I have ideas for how we can reuse just a single homepage but not sure when I'll get to it.

Also probably beneficial to use FlashList both here and for the recents section — the lists are small but that way we limit it to a few items rendered at a time.